metals/src/main/scala/scala/meta/internal/metals/BuildServerConnection.scala

```diff
@@ -305,6 +305,30 @@ class BuildServerConnection private (
     }
   }
 
+  def jvmTestEnvironment(
```

Comment:
This uses the non-deprecated api from the bsp spec, but bazel-bsp doesn't seem to be responding to this correctly (even though it supposedly supports it) so I'm leaving this here for if/when it does start working.